Misc/stable_abi.toml

```diff
@@ -1592,8 +1592,6 @@
     added = '3.2'
 [data.PyUnicode_Type]
     added = '3.2'
-[function.PyWeakref_GetObject]
-    added = '3.2'
```

Comment:
We need to leave this here. It probably has to keep behaving the same way, too.

However, it doesn't need to appear in header files anymore (though it should if someone specifies a version for `Py_LIMITED_API` that precedes 3.15). But we can't take it away from working code.